docs/README.md

```diff
@@ -1,37 +1,17 @@
-# Documentation Index
+# Documentation
 
-## Getting started
+## User Documentation
 
-* [Frequently Asked Questions](frequently_asked_questions.md)
-* Quickstart guides
-  * [AWS](quickstart-aws.md)
-  * [Azure](quickstart-azure.md)
-  * [DigitalOcean](quickstart-digitalocean.md)
-  * [Google Compute Engine (GCE)](quickstart-gce.md)
-  * [Hetzner](quickstart-hetzner.md)
-  * [OpenStack](quickstart-openstack.md)
-  * [Packet](quickstart-packet.md)
-  * [VMware vSphere](quickstart-vsphere.md)
+KubeOne user documentation can be found here: https://docs.loodse.com/kubeone/
 
-## Using KubeOne
-
-* [SSH requirements](ssh.md)
-* [Upgrading Kubernetes Clusters Using KubeOne](upgrading_cluster.md)
-* [Using Addons](addons.md)
-* [Proxy support](proxy.md)
-* [Manual Cluster Repair](manual-cluster-repair.md)
-* [Environment variables used by KubeOne](environment_variables.md)
-* [Terraform LoadBalancers in Examples](example-loadbalancer.md)
-* [Migrating to the KubeOneCluster API](api_migration.md)
-
-## Development
+## Development Documentation
 
 * [Adding support for a provider](adding_provider_support.md)
 * [Project Structure](project_structure.md)
```

Comment:
You've removed this document, so we should remove it from the index or restore the document. Since it's very outdated, I'd say remove it for now and rewrite it once we have a more stable structure.